backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -281,13 +281,16 @@ public static Boolean parseYesNo(String v) {
           "MS", "MT", "NC", "ND", "NE", "NH", "NJ", "NM", "NV", "NY", "OH", "OK", "OR", "PA", "PR",
           "PW", "RI", "SC", "SD", "TN", "TX", "UT", "VA", "VI", "VT", "WA", "WI", "WV", "WY", "NA");
 
+  private static final Set<String> CANADIAN_STATE_CODES =
+      Set.of("AB", "BC", "MB", "NB", "NL", "NT", "NS", "NU", "ON", "PE", "QC", "SK", "YT");
+
   public static String parseState(String s) {
     String state = parseString(s);
     if (state == null) {
       return null;
     }
     state = state.toUpperCase();
-    if (STATE_CODES.contains(state)) {
+    if (STATE_CODES.contains(state) || CANADIAN_STATE_CODES.contains(state)) {
```

Comment:
nit: my brain did only hang for a half second on this, but for the sake of improving readability by a tiny bit/consistency with the new constant, would consider renaming STATE_CODES --> US_STATE_CODES

Reply:
I would normally agree but I think we do use it in a couple other places, so for the sake of getting this fix in I'm going to keep it as-is. A good idea to change in the future though!